Shruti Bist/Assignment-2/basic-auth/blog/templates/blogs.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/blog.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <title>Blogs</title>
+</head>
+<body>
+    
+   
+    <div class="has-bg-img my=0 py-0" style="background-image: url(https://source.unsplash.com/-5fbmfaInwg/1600x900
+    ); "><div class="container">
+        <h1 class="py-5 text-end">My Blogs Website</h1>
+
+        <h2 class="mt-4 pt-4 text-end ">Come together with us</h2>
+         
+        
+        <h5 class="pb-5 pt-3 text-end ">Share your stories through our website</h5>
+        <br>
+      </div>
+    </div>
+
+    <div class="new py-4 my-4 d-grid gap-2 col-6 mx-auto">
+        <a role="button" href="create" class="btn btn-dark" >Create New blog</a>
+         </div>
+    
+
+
+    <div class="container-fluid">
+        {% for blog in blogs %}
+        <div class="card text-center bg-light" >
+            <div class="card-body">
+              <h5 class="card-title">{{blog.title}}</h5>
+              
+              <p class="card-text">{{blog.description}}.</p>
+              <a href="update/{{blog.id}}" class="btn btn-primary" role="button"> update this blog</a>
```

Comment:
Not the right way for using href , use {% url 'update-blog' blog.id %}
for both update and delete